dotnet-ory-network/README.md

```diff
@@ -0,0 +1,58 @@
+# Secure an ASP.NET Core application using Ory Network
+
+This repo demonstrates how you can use Ory Network or Ory Kratos with ASP.NET
+Core apps. This app is not for production use and serves as an example of
+integration.
+
+## Overview
+
+Generated using the default `dotnet new mvc` command.
```

Comment:
Is there a way to easily distinguish all the generated files? Could we move them all to one directory, maybe?